.github/workflows/publish.yml

```diff
@@ -0,0 +1,45 @@
+name: Publish to PyPI
+
+on: ["push", "pull_request"]
+
+jobs:
+  packages:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.x
+
+    - name: Get tags
+      run: git fetch --depth=1 origin +refs/tags/*:refs/tags/*
```

Comment:
This will fail b/c there are no tags. Once we issue a first release this will one will pass.